cfgov/alerts/tests/test_github_alert.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+from django.test import TestCase
+from mock import patch
+
+import github3
+
+from alerts.github_alert import GithubAlert
+
+
+class TestGithubAlert(TestCase):
+
+    closed_issue = github3.issues.issue.Issue(
```

Comment:
FYI, after our conversation yesterday around mocking I came across [this part](http://github3py.readthedocs.io/en/latest/testing.html#example-testing-the-release-object) of the github3 library's documentation which talks about a recommended way of mocking.

Reply:
I had seen that too, but didn't look too closely.  Was there anything specific from that resource that you think should inform the design here?